assets/constants.ts

```diff
@@ -53,6 +53,8 @@ export = {
     "story": ["#F8BF74", "#D68F78", "#8EBBB6", "#B3D8F1"]
   },
 
+  "ALLOWED_TOPIC_CATEGORIES": ["Mathematics"],
```

Comment:
Shouldn't the category be 'math' since the classroom/subject name is 'math'. ([ref](https://github.com/oppia/oppia/blob/3b89e1f6340586f288267322246be12e66653bb9/feconf.py#L976))

Reply:
@Showtim3

Reply:
Hey, @iamprayush I am not sure about this either, I have started an email thread with you, Akshay, Kevin, and Sean. WIll update the conclusion here too.